docs/pages/kubernetes-access/helm/reference.mdx

```diff
@@ -1480,6 +1480,64 @@ You can specify multiple apps by adding additional list elements.
   You can see a list of all the supported [values which can be used in a Teleport application access configuration here](../../application-access/reference.mdx#configuration).
 </Admonition>
 
+## `awsDatabases`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `list` | `[]` | When `db` chart role is used |
+
+`awsDatabases` is a YAML list object detailing the filters for the AWS databases that should be discovered and proxied by Teleport Database access.
+
+You can specify multiple database filters by adding additional list elements.
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  awsDatabases:
+    - types: ["rds", "aurora"]
```

Comment:
currently the `rds` type will discover both RDS dbs and Aurora clusters (no type `aurora`). `redshift` will likely come in 9.0.

https://github.com/gravitational/teleport/blob/538fcaa98007266e7940df51ca6a2e584cf86be1/lib/services/matchers.go#L60-L65

Reply:
Fixed, thanks!